### PR TITLE
Create a confirm commit task-plugin

### PR DESCRIPTION
**Pull request by ktbyers**
Created a separate napalm_confirm_commit task plugin as I thought it was more logical (than having this functionality embedded in napalm_configure).

Having confirm_commit in napalm_configure would create strange behaviors depending on which set of arguments were passed in (or we would need to add a bunch of argument checks which I thought was a bad idea).

I will fix the commit-confirm/revert_in PR after this PR is merged.

### CODE REVIEW

**Comment by ktbyers** [comment]
This PR is related to:

https://github.com/nornir-automation/nornir_napalm/pull/44

**Comment by ktbyers** [comment]
@dbarrosop Are you okay with me merging this? I think it should be good-to-go.

File: nornir_napalm/plugins/tasks/napalm_confirm_commit.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+from nornir.core.task import Result, Task
+
+from nornir_napalm.plugins.connections import CONNECTION_NAME
+
+
+def napalm_confirm_commit(task: Task, dry_run: Optional[bool] = None) -> Result:
```
Comment (dbarrosop) [inline nornir_napalm/plugins/tasks/napalm_confirm_commit.py:7]: Just a minor suggestion (feel free to ignore). I think `dry_run: Optional[bool] = None` could be `dry_run: bool = False` simplifying slightly the signature and having no impact on the code.

Comment (dbarrosop) [reply nornir_napalm/plugins/tasks/napalm_confirm_commit.py:7]: disregard, consistency with others...

**Review by dbarrosop — APPROVED** [review]

**Comment by dbarrosop** [comment]
Yes, this looks good to me!

**Pull request merged**